mpop_service/logutils.h

```diff
@@ -0,0 +1,53 @@
+#ifndef LOGUTILS_H
+#define LOGUTILS_H
+
+
+#define LOGSIZE 1024 * 100 //log size in bytes
+#define LOGFILES 5 // keep recent 5 logs file
+
+#include <QObject>
+#include <QString>
+#include <QDebug>
+#include <QDate>
+#include <QTime>
+
+/**
+ * @brief The Logutils class implements logs logic for this application.
+ *
+ * Different API can call the methods of the single instance of this object, if needed.
+ */
+class logutils : public QObject
```

Comment:
Class names should start with a capital letter. I suggest LogUtils - or Logger, actually.